RJL.UIP.HW3.ConditionalsMethods/Program.cs

```diff
@@ -0,0 +1,245 @@
+            //int randomIntFactorialTask2 = 30;
+            double resultFactorialTask2 = GetFactorial(randomIntFactorialTask2);
+            Console.WriteLine("task2");
+            Console.WriteLine($"Factorial of number {randomIntFactorialTask2} = {resultFactorialTask2}");
+            Console.WriteLine("-------------------------------------------------------------");
+            // task3
+            int randomIntFibonachiTask3 = randomizer.Next(1, 10);
+            int[] resultFibonachiTask3 = new int[randomIntFibonachiTask3];
+            Console.WriteLine("task3");
+            Console.Write($"Fibonacci sequence with length {randomIntFibonachiTask3} = ");
+            for (int i = resultFibonachiTask3.Length - 1; i >= 0; i--)
+            {
+                resultFibonachiTask3[i] = GetFibonachi(randomIntFibonachiTask3 - 1 - i);
+                Console.Write(" " + resultFibonachiTask3[i]);
+            }
+            Console.WriteLine();
+            Console.WriteLine("-------------------------------------------------------------");
+            //task4
+            int inputYearValueTask4;
+            int inputMonthValueTask4;
+            int resultCountDaysTask4;
+            Console.WriteLine("Task4");
+            Console.WriteLine("Enter value for Year");
+            string inputStringYear = Console.ReadLine();
+            bool isSuccessYear = int.TryParse(inputStringYear, out inputYearValueTask4);
+            Console.WriteLine("Enter value for Month");
+            string inputStringMonth = Console.ReadLine();
+            bool isSuccessMonth = int.TryParse(inputStringMonth, out inputMonthValueTask4);
+            resultCountDaysTask4 = GetCountDays(inputYearValueTask4, inputMonthValueTask4);
+            Console.WriteLine($"{inputYearValueTask4} year {inputMonthValueTask4} month contains {resultCountDaysTask4} days");
+            Console.WriteLine("-------------------------------------------------------------");
+            // task5
+            Console.WriteLine("Task5");
+            int[] randomArrayTask5 = CreateRandomArray();
+            Console.WriteLine("Random current array is");
+            PrintArray(randomArrayTask5);
+            int[] sortArrayTask5 = randomArrayTask5;
+            QuickSortArray(sortArrayTask5, 0, randomArrayTask5.Length - 1);
+            Console.WriteLine();
+            Console.WriteLine("Sorted array is ");
+            PrintArray(sortArrayTask5);
+            Console.WriteLine();
+            Console.WriteLine("-------------------------------------------------------------");
+            //task6
+            Console.WriteLine("Task6");
+            int[] randomArrayTask6 = CreateRandomArray();
+            Console.WriteLine("Random current array is");
+            PrintArray(randomArrayTask6);
+            Console.WriteLine();
+            SetNullMinMaxArray(randomArrayTask6);
+            Console.WriteLine("Array with null value between min and max is");
```

Comment:
As as uset I don't understand how does it works, you should add some explanations like "a min value is N and the index is M..."etc
Now I see this and I don't understand
_Task6
Random current array is
 3 8 2 6 1 6 8 7 2 8 8 0 4 1 3 2 5 7 7 3 0 0 8 0 3 5 5 4 5 6 6 1 1 7 4 2 2 0 7 8
Array with null value between min and max is
 3 8 0 0 0 0 0 0 0 0 0 0 4 1 3 2 5 7 7 3 0 0 8 0 3 5 5 4 5 6 6 1 1 7 4 2 2 0 7 8_